test/configuration.js

```diff
@@ -167,6 +167,33 @@ describe('configuration', function () {
       chai.config.showDiff = !chai.config.showDiff;
       assert.equal(chai.Assertion.showDiff, chai.config.showDiff);
     });
-    
+  });
+
+  describe('useProxy', function() {
+    var readNoExistentProperty = function() {
+      expect(false).to.be.tue; // typo: tue should be true
+    };
+
+    it('should have default value equal to true', function() {
+      expect(chai.config.useProxy).to.be.true;
+    });
+
+    describe('when true', function() {
+      it('should use proxy unless user\'s environment doesn\'t support', function() {
+        if (typeof Proxy !== 'undefined' || typeof Reflect !== 'undefined') {
```

Comment:
@lucasfcosta I'm in favor of keeping it the way it is. I prefer achieving complete test coverage by testing in different environments and browsers rather than shimming stuff or whatever the opposite of shimming would be by setting a built-in to `undefined`.